runtime/doc/lsp.txt

```diff
@@ -1458,17 +1458,18 @@ extract_completion_items({result})
                     https://microsoft.github.io/language-server-protocol/specification#textDocument_completion
 
 get_effective_tabstop({bufnr})          *vim.lsp.util.get_effective_tabstop()*
-                Returns visual width of tabstop.
+                Returns indentation size.
```

Comment:
Hmm, this is generated from the doxygen comments.

Reply:
@tpope line 1901 and co need to get updated, then if you re-run the document generation pipeline it should overwrite the manual changes.

Reply:
Good catch, I'll fix that.